src/transliteration/ui/LineGroup.ts

```diff
@@ -12,9 +12,24 @@ export interface LineInfo {
   textService: TextService
 }
 
+function createLineTokens(
```

Comment:
Extracts aligned tokens, including the omitted ones. The order of the result doesn't matter as it's sorted later on.